src/frontends/pytorch/src/op/quantize.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/clamp.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/round.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils_quantize.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_quantize_per_tensor(const NodeContext& context) {
+    num_inputs_check(context, 4, 4);
+    const auto input = context.get_input(0);
+    const auto scale = context.get_input(1);
+    const auto zero_point = context.get_input(2);
+    const auto dtype = convert_dtype(context.const_input<int64_t>(3));
+
+    const auto scale_convert = context.mark_node(std::make_shared<v1::ConvertLike>(scale, input));
+    const auto zero_point_convert = context.mark_node(std::make_shared<v1::ConvertLike>(zero_point, input));
+    const auto scaled_input = context.mark_node(std::make_shared<v1::Divide>(input, scale_convert));
+    const auto scaled_input_with_zero_pt =
+        context.mark_node(std::make_shared<v1::Add>(scaled_input, zero_point_convert));
+    const auto quantized_input =
+        context.mark_node(std::make_shared<v5::Round>(scaled_input_with_zero_pt, v5::Round::RoundMode::HALF_TO_EVEN));
+
+    ov::Output<ov::Node> output;
+    if (dtype == element::u8) {
+        const auto clamp = context.mark_node(std::make_shared<v0::Clamp>(quantized_input,
```

Comment:
As we discussed this operation should be implemented using FakeQuantize. To understand how arguments of quantize_per_XXX operation translate to arguments of openvino FakeQuantize you can look how pytorch fake_quantize_per_XXX operation was implemented in frontend.

Reply:
Implemented!